client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/styles.scss

```diff
@@ -418,4 +413,10 @@
 		width: 150px;
 		@include placeholder();
 	}
+
+	button.components-button.bulk-domain-transfer__cta {
+		&:disabled {
+			background: var(--studio-gray-5);
+		}
+	}
```

Comment:
Can we move this over to the wp-components-overrides file and generalize for all buttons? or if that's not an option right now at least into the stepper global.scss?